roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -28,9 +27,9 @@
         continue
     if not aliases.is_file():
         continue
-    if targets_from_cli and target.name not in targets_from_cli:
-        continue
     lines = aliases.read_text().split("\n")
+    if targets_from_cli and not {*lines, target.name} & {*targets_from_cli}:
```

Comment:
I would suggest to write `set(targets_from_cli)` instead of `{*targets_from_cli}`. This just because I'm more familiar with this syntax.